CHANGELOG.md

```diff
@@ -5,6 +5,86 @@
 <!-- For unreleased changes, see entries in .chloggen -->
 <!-- next version -->
 
+## v0.140.0
+
+This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.140.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.140.0)
+and the [opentelemetry-collector-contrib v0.140.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.140.1) releases where appropriate.
+
+### 🛑 Breaking changes 🛑
+
+- (Splunk) `ecs_task_observer`: Removed discontinued `ecs_task_observer` extension. ([#6936](https://github.com/signalfx/splunk-otel-collector/pull/6936))
+  The `ecs_task_observer` extension has been discontinued and is no longer available in the OpenTelemetry Collector Contrib repository
+  and was removed via [PR 43818](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/43818).
+- (Splunk) `gateway`: Remove SignalFx receiver from the default gateway config in favor of the HTTP forwarder extension ([#6887](https://github.com/signalfx/splunk-otel-collector/pull/6887))
+  This change means that metrics and logs sent to the gateway in the SignalFx format
+  will no longer be subject to the batch and memory limiter processors configured in
+  the gateway pipelines. Please ensure any required memory limiting or batching is
+  configured on the sending side (e.g. the agent). Please note that the `signalfx` receiver
+  will eventually be deprecated and removed.
+
+- (Contrib) `all`: Latest supported k8s version is moved from 1.17 to 1.21. ([#43891](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/43891))
+- (Contrib) `cmd/otelcontribcol`: Removing unmaintained component extension/ecstaskobserver ([#43818](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/43818))
```

Comment:
Not a huge deal, but this is the same entry as `(Splunk) ecs_task_observer` above.